ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -905,92 +914,32 @@ func TestCheckNetworkBlackHolePort(t *testing.T) {
 	testNetworkFaultInjectionCommon(t, tcs, types.BlackHolePortFaultType, http.MethodGet)
 }
 
-func generateNetworkLatencyTestCases(name, expectedHappyResponseBody string) []networkFaultInjectionTestCase {
-	happyNetworkLatencyReqBody := map[string]interface{}{
-		"DelayMilliseconds":  delayMilliseconds,
-		"JitterMilliseconds": jitterMilliseconds,
-		"Sources":            ipSources,
-	}
+func generateCommonNetworkLatencyTestCases(name string) []networkFaultInjectionTestCase {
 	tcs := []networkFaultInjectionTestCase{
-		{
-			name:                 fmt.Sprintf("%s success", name),
-			expectedStatusCode:   200,
-			requestBody:          happyNetworkLatencyReqBody,
-			expectedResponseBody: types.NewNetworkFaultInjectionSuccessResponse(expectedHappyResponseBody),
-			setAgentStateExpectations: func(agentState *mock_state.MockAgentState) {
-				agentState.EXPECT().GetTaskMetadata(endpointId).
-					Return(happyTaskResponse, nil).
-					Times(1)
-			},
-		},
-		{
-			name:               fmt.Sprintf("%s unknown request body", name),
-			expectedStatusCode: 200,
-			requestBody: map[string]interface{}{
-				"DelayMilliseconds":  delayMilliseconds,
-				"JitterMilliseconds": jitterMilliseconds,
-				"Sources":            ipSources,
-				"Unknown":            "",
-			},
-			expectedResponseBody: types.NewNetworkFaultInjectionSuccessResponse(expectedHappyResponseBody),
-			setAgentStateExpectations: func(agentState *mock_state.MockAgentState) {
-				agentState.EXPECT().GetTaskMetadata(endpointId).
-					Return(happyTaskResponse, nil).
-					Times(1)
-			},
-		},
-		{
-			name:                 fmt.Sprintf("%s no request body", name),
-			expectedStatusCode:   400,
-			requestBody:          nil,
-			expectedResponseBody: types.NewNetworkFaultInjectionErrorResponse("required request body is missing"),
-			setAgentStateExpectations: func(agentState *mock_state.MockAgentState) {
-				agentState.EXPECT().GetTaskMetadata(endpointId).Return(state.TaskResponse{}, nil).Times(0)
-			},
-		},
 		{
 			name:               fmt.Sprintf("%s malformed request body 1", name),
 			expectedStatusCode: 400,
 			requestBody: map[string]interface{}{
 				"DelayMilliseconds":  "incorrect-field",
-				"JitterMilliseconds": jitterMilliseconds,
+				"JitterMilliseconds": "incorrect-field",
```

Comment:
I think this is intentional that only one of them is incorrect.

Reply:
Got it. Changing it back in the next revision